packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -103,18 +103,24 @@ const LineItemMeta = styled.div< { theme?: Theme } >`
 	color: ${ ( props ) => props.theme.colors.textColorLight };
 	font-size: 14px;
 	width: 100%;
+	display: flex;
+	flex-direction: row;
+	align-content: center;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	> {
```

Comment:
For my own edification, what does this `> {` and the following block mean, exactly? Is it applying spacing between the children of this div? If so, could the same thing be accomplished with the flexbox `gap` property?

Reply:
Yes, both `<NotApplicableCallout />` and `<DiscountCallout />` had small top margins that caused them to be misaligned when on the same row, which the direct child selector replaced. However, using `gap` is a much more elegant solution because it prevents an unnecessary top margin for only a single row. I will go ahead and switch to `gap`!